samples/tls/logging/README.md

```diff
@@ -7,20 +7,19 @@ TLS enablement for SAS Viya logging is divided into two parts:
 - **TLS for in-cluster communications**, which is between logging components within
 the cluster (between the Elasticsearch nodes and between Elasticsearch and other logging components). TLS must be enabled on these connections.
 
-- **TLS for communications into the cluster**, which is between a user's 
-browser and Kibana. TLS is optional on these connections, although enabling
-TLS is a best practice.
+In-cluster communications must always take place through TLS-enabled connections. TLS for in-cluster communications is configured using a separate process, 
+either automatically (using cert-manager to generate certificates) 
+or by manually generating certificates. See [Notes_on_using_TLS](../../../logging/Notes_on_using_TLS.md) for information about configuring TLS for 
+in-cluster communications.
+
+- **TLS for communications into the cluster**, which is between Ingress or a 
+user's browser (if NodePorts are used) and Kibana. TLS is optional on these connections, although enabling TLS is a best practice.
 
```

Comment:
Revise the summary for the corresponding section (one TLS for communications into the cluster) in NOTE ON USING TLS to match this one.